etc/proto/test_no_changes.sh

```diff
@@ -28,8 +37,22 @@ find src -regex ".*\.pb\.go" \
   | awk '{print $1}'
 )"
 
+# hash newly generated python code
```

Comment:
Did you try modifying a proto and making sure this would trip?

Reply:
yes, this tripped when i tested it because the python protos did indeed need to be regenerated (see the pfs change included in this PR)